cg/store/api/find_business_data.py

```diff
@@ -330,6 +330,16 @@ def get_number_of_reads_for_sample_passing_q30_threshold(
         reads_count: Optional[int] = total_reads_query.scalar()
         return reads_count if reads_count else 0
 
+    def get_sample_lane_sequencing_metrics_by_flow_cell_name(
```

Comment:
This looks a bit off topic for the PR since it is not used. Can you extract it to another PR? (same for its tests and filters)

Reply:
I added these functions since in a test sql alchemy syntax was used outside of filter functions. We spent a lot of time prior in this year to not have that anymore. I can extract it if needed, on the other hand it is implemented and tested in this pr now.

Are you sure an additional PR is needed for this?

Reply:
https://github.com/Clinical-Genomics/cg/pull/2253